containerz/containerz.proto

```diff
@@ -619,6 +619,11 @@ message LocalDriverOptions {
   string mountpoint = 3;
 }
 
+message CustomOptions {
```

Comment:
Should we describe here what the "custom" driver is? Is there a documentation reference point for this?

(I'm also curious whether this 'custom' driver is something that we're expecting to be supported across different container runtimes.)

Reply:
There is no one custom driver - this set of options applies to any third party volume plugin. The container manager simply passes these options to the plugin/driver.

Reply:
LGTM, thanks for adding the comment.